lab_4_fill_words_by_ngrams/main.py

```diff
@@ -114,6 +177,9 @@ def __init__(self) -> None:
         """
         Initialize an instance of GeneratorTypes.
         """
+        self.greedy = 0
```

Comment:
where do you use it?

Reply:
In start.py (line 27). Added to get_conversion_generator_type() also for clarity